### PR TITLE
Lint imports

### PR DESCRIPTION
**Pull request by thewilkybarkid**

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -1,2 +1,4 @@
 [flake8]
+application-import-names = content_store
+import-order-style = google
```
Comment (giorgiosironi) [inline .flake8]: roughly standard library, dependencies, current codebase?

File: Pipfile.lock [hunk 2]
```diff
@@ -109,13 +109,21 @@
...
+            "version": "==0.18"
+        },
         "isort": {
             "hashes": [
                 "sha256:1153601da39a25b14ddc54955dbbacbb6b2d19135386699e2ad58517953b34af",
                 "sha256:b9c40e9750f3d77e6e4d441d8b0266cf555e7cdabdcff33c4fd06366ca761ef8",
                 "sha256:ec9ef8f4a9bc6f71eec99e1806bfa2de401650d996c59330782b89a5555c1497"
             ],
-            "markers": "python_version != '3.0.*' and python_version != '3.1.*' and python_version != '3.2.*' and python_version >= '2.7' and python_version != '3.3.*'",
+            "markers": "python_version != '3.1.*' and python_version != '3.0.*' and python_version >= '2.7' and python_version != '3.2.*' and python_version != '3.3.*'",
```
Comment (giorgiosironi) [inline Pipfile.lock]: was this locked using the `README` guide and it changed anyway?

Comment (giorgiosironi) [reply Pipfile.lock]: The values are identical, it's only the order in this case that changed: https://github.com/pypa/pipenv/issues/2760

Comment (thewilkybarkid) [reply Pipfile.lock]: Ah, hadn’t seen `pipenv lock`. Need to look at what that does given the lock file is already updated.

**Comment by thewilkybarkid** [comment]
Closing this rather than resolving the conflict, as the repo's about to be archived.

**Pull request closed without merging**